tests/e2e/specs/settings.spec.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * External dependencies
+ */
+const { test, expect } = require( '@playwright/test' );
+
+/**
+ * Internal dependencies
+ */
+import SettingPage from '../utils/pages/settings.js';
+import ElementLocators from '../utils/element-locators.js';
+
+/**
+ * @type {import('../utils/pages/settings.js').default} settingPage
+ */
+let settingPage = null;
+
+/**
+ * @type {import('../utils/element-locators.js').default} onboardingPage
+ */
+let locator = null;
+
+/**
+ * @type {import('@playwright/test').Page} page
+ */
+let page = null;
+
+test.describe( 'Snapchat Settings', () => {
+	test.use( { storageState: process.env.ADMINSTATE } );
+
+	test.beforeAll( async ( { browser } ) => {
+		page = await browser.newPage();
+		settingPage = new SettingPage( page );
+		locator = new ElementLocators( page );
+
+		await settingPage.mockJetpackConnected();
+		await settingPage.mockSnapchatConnection( { status: 'connected' } );
+		await settingPage.mockOnboardingSetup( {
+			status: 'connected',
+			step: 'accounts',
+		} );
+	} );
+
+	test.afterAll( async () => {
+		await settingPage.closePage();
+	} );
```

Comment:
To speed up these tests, can we avoid closing and reloading the page between each test and instead only reload the page in tests where it's needed?

Reply:
> can we avoid closing and reloading the page between each test

We're opening the page once with `beforeAll()`, and closing the page once with `afterAll()`, so it closes after all the tests are completed within a file.